cmd/root.go

```diff
@@ -151,6 +151,7 @@ func onInit(cmd *cobra.Command, args []string) error {
 // initialize anything related to logging, metrics, or tracing
 func initObservation() {
 	logrusInstance := logrus.New()
+	logrusInstance.SetOutput(os.Stderr)
```

Comment:
What's the change here?
We're building some tooling around DCE CLI that relies on the logging output, so I just want to make sure we call out any breaking changes there.
(I'd thought current behavior was already pushing everything to stderr?)

Reply:
Made note of in the CHANGELOG.